renderdoc/driver/vulkan/vk_core.h

```diff
@@ -70,6 +70,32 @@ enum class VkIndirectPatchType
   DrawIndirectByteCount,
 };
 
+enum class EventFlags : uint32_t
+{
+  NoFlags = 0x0,
+  VertexPipeRWUsage = 1 << uint32_t(ShaderStage::Vertex),
+  HullPipeRWUsage = 1 << uint32_t(ShaderStage::Hull),
```

Comment:
Slight nitpick - 'hull pipe' doesn't have a good meaning. 'vertex pipe' refers to all the shader stages before rasterization, so I think we can drop the `Pipe` from all of these variable names (and maybe drop the non-vulkan aliases like Domain and Hull).

Reply:
Makes sense, done.